test/opf.jl

```diff
@@ -217,6 +217,35 @@ end
     end
 end
 
+@testset "test qc opf with trilinear convexhull relaxation" begin
+    @testset "3-bus case" begin
+        result = run_opf("../test/data/case3.m", QCWRTriPowerModel, ipopt_solver)
+
+        @test result["status"] == :LocalOptimal
+        @test isapprox(result["objective"], 5817.58; atol = 1e0)
+    end
+    @testset "5-bus asymmetric case" begin
+        result = run_opf("../test/data/case5_asym.m", QCWRTriPowerModel, ipopt_solver)
+
+        @test result["status"] == :LocalOptimal
+        @test isapprox(result["objective"], 15816.9; atol = 1e0)
+    end
+    @testset "6-bus case" begin
+        result = run_opf("../test/data/case6.m", QCWRTriPowerModel, ipopt_solver)
+
+        @test result["status"] == :LocalOptimal
+        @test isapprox(result["objective"], 11567.1; atol = 1e0)
+        @test isapprox(result["solution"]["bus"]["1"]["va"], 0.0; atol = 1e-4)
+        @test isapprox(result["solution"]["bus"]["4"]["va"], 0.0; atol = 1e-4)
+    end
+    @testset "24-bus rts case" begin
+        result = run_opf("../test/data/case24.m", QCWRTriPowerModel, ipopt_solver)
+
+        @test result["status"] == :UserLimit
```

Comment:
Seems strange this would be hitting the user limit.  Maybe an indication of a bug.  If not a bug, in the interest of the test runtime, it is better to omit this test.